.github/workflows/update-tor.yml

```
@@ -0,0 +1,110 @@
+name: Update TOR binaries
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-tor:
+    runs-on: ubuntu-latest
+    env:
+      UPDATE_FILE: packages/suite-data/files/bin/tor/update.sh
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Get current TOR versions
```

```suggestion
      - name: Get current TOR versions
        id: versions
        run: |
          crx_next=$(grep "^CRX_VER=" $UPDATE_FILE | cut -d'=' -f2)
          arm_next=$(grep "^CRX_LINUX_ARM_VER=" $UPDATE_FILE | cut -d'=' -f2)
          
          echo "crx_next=$crx_next" >> $GITHUB_OUTPUT
          echo "arm_next=$arm_next" >> $GITHUB_OUTPUT
      
```
If we want to extract the version number, we can simply do it like this.

---

This doesn't increase the patch number by one. (Fair, non-descriptive function name by me.)

---

Okay, I understand now. I don't see any versioning that would coincide with the CRX versions we use. Brave basically increments it with the Dynamo DB table upon uploading the built CRX package.